Bottom: d17dae60f494db83b6001b0620974fbf2e9f0306
Top:    dce1463667b1c6ad3ccef0d7cc5f651ff6d0f3fa
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 15:55:33 -0500

Enable change hooks from Gitlab


---
diff --git a/docker/buildbot-master/buildbot/master.cfg b/docker/buildbot-master/buildbot/master.cfg
index bc7e7ccd..02056a49 100644
--- a/docker/buildbot-master/buildbot/master.cfg
+++ b/docker/buildbot-master/buildbot/master.cfg
@@ -91,7 +91,12 @@ c['www'] = {
                 ]),
             ],
         ),
+    'change_hook_dialects': {
+        'gitlab': {
+            'secret': os.getenv('GITLAB_WEBHOOK_SECRET')
+        }
     }
+}
 
 ####### PROJECT IDENTITY
